WizardStyle.css

```diff
@@ -41,3 +41,11 @@ details {
 summary, .setAll {
   background-color: #F56600;
 }
+
+li {
```

Comment:
This change affects other pages on the site that look better without the extra spacing. Need to create a class to select the elements that need this spacing. Also, is there a way to apply this to the <ol> rather than each <li> to save an excessive number of style attributes?

Reply:
Class is probably best

Reply:
I wonder whether setting line-height might work better. Go for whatever works.